Formula/e/envoy.rb

```diff
@@ -64,13 +71,11 @@ def install
       # https://github.com/envoyproxy/envoy/issues/37911
       args << "--copt=-Wno-thread-safety-reference-return"
 
-      # Workaround to build with Clang 19 until envoy uses newer tcmalloc
-      # https://github.com/google/tcmalloc/commit/a37da0243b83bd2a7b1b53c187efd4fbf46e6e38
-      args << "--copt=-Wno-unused-but-set-variable"
-
-      # Workaround to build with Clang 19 until envoy uses newer grpc
-      # https://github.com/grpc/grpc/commit/e55f69cedd0ef7344e0bcb64b5ec9205e6aa4f04
-      args << "--copt=-Wno-missing-template-arg-list-after-template-kw"
```

Comment:
not sure why, but it seems this needs to be added for macos build as I see in the error on CI? 

```
  Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
  In file included from external/com_github_grpc_grpc/src/core/lib/resource_quota/memory_quota.cc:43:
  In file included from external/com_github_grpc_grpc/src/core/lib/promise/seq.h:25:
  external/com_github_grpc_grpc/src/core/lib/promise/detail/basic_seq.h:103:38: error: a template argument list is expected after a name prefixed by the template keyword [-Wmissing-template-arg-list-after-template-kw]
    103 |                     Traits::template CallSeqFactory(f_, *cur_, std::move(arg)));
        |                                      ^
```